LICENSE-HEADER

```diff
@@ -0,0 +1,2 @@
+Copyright contributors to the kepler.gl project
+SPDX-License-Identifier: MIT
```

Comment:
Note that order matters due to an assumption made in [uber-licence](https://github.com/uber/uber-licence/blob/e7a9dc5058201e1ddc8f5d4d4ebcaca66a755b0f/license-fixer.js#L53)

Reply:
Forked it over, removed the parts causing issues, and made other necessary changes to get it to work (like ES modules and deps)